File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/jsonviewer/adapter/BaseJsonViewerAdapter.java
package com.didichuxing.doraemonkit.widget.jsonviewer.adapter;


import androidx.recyclerview.widget.RecyclerView;

/**
 * Created by yuyuhang on 2017/11/30.
 */
public abstract class BaseJsonViewerAdapter<VH extends RecyclerView.ViewHolder> extends RecyclerView.Adapter<VH> {

    public static int KEY_COLOR = 0xFF922799;
    public static int TEXT_COLOR = 0xFF3AB54A;
    public static int NUMBER_COLOR = 0xFF25AAE2;
    public static int BOOLEAN_COLOR = 0xFFF98280;
    public static int URL_COLOR = 0xFF66D2D5;
    public static int NULL_COLOR = 0xFFEF5935;
    public static int BRACES_COLOR = 0xFF4A555F;

    public static float TEXT_SIZE_DP = 12;
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/jsonviewer/adapter/BaseJsonViewerAdapter.kt
package com.didichuxing.doraemonkit.widget.jsonviewer.adapter

import androidx.recyclerview.widget.RecyclerView

/**
 * Created by yuyuhang on 2017/11/30.
 */
abstract class BaseJsonViewerAdapter<VH : RecyclerView.ViewHolder?> : RecyclerView.Adapter<VH>() {
    companion object {
        var KEY_COLOR = -0x6dd867
        var TEXT_COLOR = -0xc54ab6
        var NUMBER_COLOR = -0xda551e
        var BOOLEAN_COLOR = -0x67d80
        var URL_COLOR = -0x992d2b
        var NULL_COLOR = -0x10a6cb
        var BRACES_COLOR = -0xb5aaa1
        var TEXT_SIZE_DP = 12f
    }
}